src/app/containers/ATIAnalytics/atiUrl/index.test.js

```diff
@@ -36,7 +36,7 @@ describe('getThingAttributes', () => {
       mockAndSet(func, null);
     });
 
-    const queryParams = atiPageViewParams({});
+    const queryParams = atiUrlBuilder.buildATIPageTrackUrl({});
```

Comment:
Tiny change, but as a style we generally do `import { buildATIPageTrackUrl } = from '.'` and use it directly :)